bot/utils.py

```diff
@@ -24,10 +21,18 @@ def format_tags(usernames):
 
 
 def parse_game(text):
-    match = re.search(GAME_REGEX, text)
-    if not match:
+    # https://regex101.com/r/9dIo49/2
+    game_regex = r'/game @([^\s]+) (\d+):(\d+) @([^\s]+)'
+    tourngame_regex = r'/tourngame @([^\s]+) (\d+):(\d+) @([^\s]+)'
```

Comment:
😢

Reply:
https://regex101.com/r/9dIo49/3

Reply:
😁

Reply:
I knew that you will solve this for me. 😊